sympy/integrals/tests/test_integrals.py

```diff
@@ -881,6 +881,16 @@ def test_as_sum_trapezoid():
     assert Integral(sign(x), (x, 0, 1)).as_sum(1, 'trapezoid') == S.Half
 
 
+def test_as_sum_simpson():
+    e = Integral((x + y)**2, (x, 0, 1))
+    assert e.as_sum(1, method="simpson").expand() == y**2 + y + Rational(1, 3)
```

Comment:
It would be better to test with `evaluate=False` and also to use an example that is not computed exactly.

Reply:
Makes sense. I am also adding a check to make sure intervals are even.

Can you please clarify why evaluate=False would be better? The tests for the other methods evaluate the summation.

Thanks!

Reply:
It would be better to test the formula that is returned rather than the result of calling doit or expand etc.

Reply:
I see your point. There is no need to do extra work that is unrelated to the test itself. I will update accordingly.

Do you think it would be worth updating the other tests in a separate pull request?